projects/ad74413r/Makefile

```diff
@@ -1,10 +1,8 @@
-# Select the example you want to enable by choosing y for enabling and n for disabling
-DUMMY_EXAMPLE = y
-IIO_EXAMPLE = n
-IIO_TRIGGER_EXAMPLE = n
-
 include ../../tools/scripts/generic_variables.mk
 
+EXAMPLE := dummy
+include ../../tools/scripts/examples.mk
```

Comment:
this is what is needed to add in a project that has examples, so basically by not adding this we can be backwards compatible with projects that don't have examples

Reply:
Can we also move these includes in the examples.mk file?
```
include ../../tools/scripts/generic_variables.mk
include src.mk
include ../../tools/scripts/generic.mk
```
Although, it may not be in the scope of this PR...

Reply:
we could, but not all projects have examples, but i agree with you it's an unnecessary repetition that we keep carrying in all projects

maybe we can do something like this in a separate PR, it's pretty easy as it only targets the src.mk files on project root folder